### ISSUE TITLE
[lxqt-panel] system tray not functional when QT_DEVICE_PIXEL_RATIO = 2

### ISSUE THREAD

**Issue by imposeren** [lxde/lxqt#914]
when I set ``QT_DEVICE_PIXEL_RATIO=2`` (for example in ``/etc/profile.d/hidpi.sh``):
* System tray icons do not respond to clicks.
* System tray icons are blurry. These may be somehow connected to ``Qt::UseHighDPIPixmaps``, ``Qt::AA_UseHighDpiPixmaps``

With default value (``QT_DEVICE_PIXEL_RATIO=1``) everything works just fine

```
lxqt-panel 0.9.0

LXQt Desktop Toolbox - Technical Info<p>
  Version:                0.10.0
  Qt:                     5.4.1

LSB Version:    core-2.0-amd64:core-2.0-noarch:core-3.0-amd64:core-3.0-noarch:core-3.1-amd64:core-3.1-noarch:core-3.2-amd64:core-3.2-noarch:core-4.0-amd64:core-4.0-noarch:core-4.1-amd64:core-4.1-noarch:security-4.0-amd64:security-4.0-noarch:security-4.1-amd64:security-4.1-noarch
Distributor ID: Ubuntu
Description:    Ubuntu 15.04
Release:        15.04
```

```
Package: lxqt-panel
Status: install ok installed
Priority: optional
Section: libs
Installed-Size: 2328
Maintainer: Julien Lavergne <gilir@ubuntu.com>
Architecture: amd64
Version: 0.9.0.0.0+bzr1010+201510230746~ubuntu15.04.1
```

**Comment by palinek** [comment]
@imposeren can you, please, test if the lxde/lxqt-panel#291 fixes the behaviour for you?

**Comment by imposeren** [comment]
tray icons now respond as expected. Only dropbox reacts strangely and sometimes does the right thing. But I think that this is not related to lxqt. Tray icons are blurry but this should be solved by #916.

Thanks for the solution
